### PR TITLE
Fix datastore, use public_key_ecdsa as primary key

### PR DESCRIPTION
**Pull request by johnnyluo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^8.0.1",
```
Comment (0xkobe) [inline package.json:3]: Is it needed?

Comment (johnnyluo) [reply package.json:3]: Not sure , it seems `npm install` add it

Comment (0xkobe) [reply package.json:3]: can you remove it? It is not necessary.

Comment (johnnyluo) [reply package.json:3]: this one is not in my change set

**Review by 0xkobe — APPROVED** [review]

**Pull request merged**